README.md

```diff
@@ -58,39 +76,13 @@ resource "libvirt_domain" "terraform_test" {
 Now you can see the plan, apply it, and then destroy the infrastructure:
 
 ```console
+$ terraform get
```

Comment:
From v0.10.0 upwards, Terraform requires `terraform init` to be run the first time. This creates a `.terraform/` directory. Personally, I've never had to use the `get` command.